experimental/uefi/proto/echo.proto

```diff
@@ -19,11 +19,11 @@ syntax = "proto3";
 package oak.experimental.uefi;
 
 message EchoRequest {
-    string message = 1;
+    bytes message = 1;
 }
 
 message EchoResponse {
-    string message = 1;
+    bytes message = 1;
 }
 
 service Echo {
```

Comment:
This is starting to look very close to `oak_functions/proto/unary_server.proto`. We could use the same proto, but leaving this untouched for now as it relates to the open question of which framing protocol we'll use.

Reply:
Yeah, we need to take a step back here and figure out how the communication is going to work. But that's a separate task.